tasks/dev.rake

```diff
@@ -7,8 +7,17 @@ Jettywrapper.url = "https://github.com/projecthydra/hydra-jetty/archive/migrate.
 RSpec::Core::RakeTask.new(:spec)
 
 desc "Run continuous integration tests"
-task ci: ['jetty:clean', 'jetty:start', 'fixtures:load', 'spec']
-
+#task ci: ['jetty:clean', 'jetty:start', 'fixtures:load', 'spec']
+task :ci do
+  ENV['environment'] = "test"
```

Comment:
I'm curious why you had to do that. What about doing this in spec_helper:
https://github.com/projecthydra/sufia/blob/master/spec/spec_helper.rb#L1